extensions/wrapper/wrapper-common-api/README.md

```diff
@@ -0,0 +1,35 @@
+<!-- PROJECT LOGO -->
+<br />
+<div align="center">
+  <a href="https://github.com/sovity/edc-extensions">
+    <img src="https://raw.githubusercontent.com/sovity/edc-ui/main/src/assets/images/sovity_logo.svg" alt="Logo" width="300">
+  </a>
+
+<h3 align="center">EDC-Connector Extension:<br />API Wrapper &amp; API Client:<br />Common API Models</h3>
+
+  <p align="center">
+    <a href="https://github.com/sovity/edc-extensions/issues/new?template=bug_report.md">Report Bug</a>
+    ·
+    <a href="https://github.com/sovity/edc-extensions/issues/new?template=feature_request.md">Request Feature</a>
+  </p>
+</div>
+
+## About this component
+
+Certain APIs to be included in our generated API Clients won't have their implementations in the wrapper extension
```

Comment:
it's not "our", it CE
Please rewrite the description in a whole, as I do not understand what this module does.